docker-compose.yml

```diff
@@ -28,6 +28,11 @@ services:
       DATABASE_NUMBER_GENERATOR_NAME:
       DATABASE_CEDAR_NAME:
       DATABASE_CIS_NAME:
+      DATABASE_ODS_HOST:
```

Comment:
I've actively chosen to include ODS_HOST separately to the other DATABASE_HOST so that we have the ability to point to different ODS instances (e.g. in staging where we connect to TEST for the CIS/CEDAR databases we might want to connect to PRE-PROD ODS)